### PR TITLE
Remove Transport.Abstractions from existence

### PR DESCRIPTION
**Pull request by davidfowl**

### CODE REVIEW

**Comment by davidfowl** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=208243

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/2546

File: src/Hosting/Hosting.sln [hunk 1]
```diff
@@ -197,7 +191,7 @@ Global
 		{02741DAE-CF50-40C7-A717-1DC7182B737C} = {2A533A99-96D8-4A02-8990-BA8F94E70567}
 		{87D20FC7-94AF-4FCF-A25F-221D61A1E2D9} = {2A533A99-96D8-4A02-8990-BA8F94E70567}
 		{28899903-4A65-466C-B8C6-DA5A5CB5DF75} = {2A533A99-96D8-4A02-8990-BA8F94E70567}
-		{006042D1-A053-4788-8D58-AB08DBB2E668} = {2A533A99-96D8-4A02-8990-BA8F94E70567}
+		{} = {2A533A99-96D8-4A02-8990-BA8F94E70567}
```
Comment (dougbu) [inline src/Hosting/Hosting.sln]:
```suggestion
```

**Review by dougbu — APPROVED** [review]
One typo

File: src/Hosting/Hosting.sln (hunk 1 above)
Comment (davidfowl) [reply src/Hosting/Hosting.sln]: I dunno if I want to force a rebuild for this change 😄

Comment (davidfowl) [reply src/Hosting/Hosting.sln]: Ah this got added somehow...

Comment (dougbu) [reply src/Hosting/Hosting.sln]: Is VS willing to open the solution without a fix here?

**Comment by anurse** [comment]
Coming in 🔥 I see...

**Comment by davidfowl** [comment]
The ci is trolling hard

**Pull request merged**